src/insight/insights.service.ts

```diff
@@ -107,4 +107,22 @@ export class InsightsService {
 
     return new PageDto(entities, pageMetaDto);
   }
+
+  async findAllFeatured(pageOptionsDto: InsightPageOptionsDto): Promise<PageDto<DbInsight>> {
+    const queryBuilder = this.insightRepository.createQueryBuilder("insights");
+
+    queryBuilder
+      .where("is_featured=true AND insights.deleted_at IS NULL")
```

Comment:
The deleted status is handled by TypeORM. 

```suggestion
      .where("is_featured=true")
```

Reply:
May want to add `.andWhere("is_public=true")` also

Reply:
When TypeORM builds the SQL query, it uses this API to build it.

Using a `where("id=1")` starts the criteria expression.

Result: `SELECT * FROM table WHERE id=1`

Using `andWhere("something_else=true")` adds to the criteria expression

Result: `SELECT * FROM table WHERE id=1 AND something_else=true`

Whenever you use a `where`, it resets the criteria expression

Using `where("id=1").andWhere("something_else=true").where("id=1")`

Result: `SELECT * FROM table WHERE id=1`

Reply:
Thanks for the detailed explanation 👍